Makefile

```diff
@@ -3,6 +3,7 @@ build:
 	rm -rf node_modules
 	npm install
 	npx gulp build
+	pipenv run ./scripts/load_templates.sh
```

Comment:
This should probably be `load-templates` so that if we need to change the load-templates process, we only have to update it in one place

Reply:
Yeah good point.